common/dynamicconfig/config_test.go

```diff
@@ -181,8 +181,33 @@ func (s *configSuite) TestUpdateConfig() {
 }
 
 func TestDynamicConfigKeyIsMapped(t *testing.T) {
-	for i := UnknownKey; i < LastKeyForTest; i++ {
-		key, ok := Keys[i]
+	for i := UnknownIntKey + 1; i < LastIntKey; i++ {
+		key, ok := IntKeys[i]
+		require.True(t, ok)
+		require.NotEmpty(t, key)
```

Comment:
since these are a real pain to identify when they fail (you just see "not true" and the line, not the iteration number): add `i`/`key` to the message?